README.md

```diff
@@ -22,6 +22,7 @@ It is required that a config.yaml exist in the root of the repository.  This wil
 Note:
 * The bucket reference name acts as an alias for referencing the bucket. If a reference name is added it must be used to reference the bucket.
 * If you are using Elasticsearch via AWS the region portion of the Elasticsearch config is required and the AWS keys are only required when the Elasticsearch Domain access policy requires keys.
+* `upperSearchResultLimit` is another optional Elasticsearch config option. It defaults to 1000 if not set. It limits the number of search results returned. We currently do not support pagination.
```

Comment:
Yeah, could you default this to something like 10000?  The only reason you would want to limit this is if you had memory or bandwidth concerns.  Every other instance it would be odd to not get documents back.